modules/openapi-generator/src/main/resources/kotlin-spring/libraries/spring-boot/gradle-wrapper.properties.mustache

```diff
@@ -0,0 +1,7 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
```

Comment:
thanks for the PR,

what about using the latest stable version 8.11.1 instead?

https://gradle.org/releases/

Reply:
Good Day.
I actually copied it from the Java templates and assumed 8.7 was selected for some reason and didn't want to upgrade unnecessarily.
Another reason is that I was considering the compatibility between Gradle and Kotlin listed here: https://kotlinlang.org/docs/gradle-configure-project.html#apply-the-plugin
When I actually look at the page, though, I see that I should have even downgraded it to 8.1.1.
Please what are your thoughts?

Reply:
> https://kotlinlang.org/docs/gradle-configure-project.html#apply-the-plugin
> When I actually look at the page, though, I see that I should have even downgraded it to 8.1.1.

Thanks for pointing it out. Let's try 8.1.1

Reply:
Okay. Done. This might not be directly related but I'm concerned with the generated samples outputting CRLF strings in various places e.g. fsharp-functions.md

Reply:
no worry. i pushed https://github.com/OpenAPITools/openapi-generator/pull/20341/commits/d0e91ba8316515febaf97f493c76a35f23d5b28f to fix it